### ISSUE TITLE
Calling 'app.get_uaa()' after creating custom UAA instance fails

### ISSUE THREAD

**Issue by oelhirai** [PredixDev/predixpy#17]
Calling `app.get_uaa()` works great after creating a UAA instance using predix.admin.app: `app.create_uaa('admin-secret')`. But after creating a custom UAA instance as such: 

``` python
uaa = predix.admin.uaa.UserAccountAuthentication(plan_name='Free')
uaa.service.name = name
uaa.create('admin-secret')
uaa.add_to_manifest(app)
```

I get a `ValueError: Instance of predix-uaa is required, create one first` when calling `app.get_uaa()` to retrieve the instance. I'm looking into this and may create a PR after.

**Comment by oelhirai** [comment]
After creating a custom UAA instance as such: 

``` python
uaa = predix.admin.uaa.UserAccountAuthentication(plan_name='Free')
uaa.service.name = name
uaa.create('admin-secret')
uaa.add_to_manifest(app)
```

I get `ValueError: Instance of predix-uaa is required, create one first` when calling `app.get_uaa()` to retrieve the instance

this call works fine when creating a UAA instance using predix.admin.app: `app.create_uaa('admin-secret')`

**Comment by j12y** [comment]
I'm adding some new unit and integration tests for UAA so will add one to test for this.

**Comment by j12y** [comment]
Should be available with `pip install predix==1.0.0rc1` which will be the next release.
